### DISCUSSION TITLE
Have to restart installation process after virtual machine is created problem.

### DISCUSSION THREAD

**Discussion by 808ScottyA** [utmapp/UTM#3445] [UTM Help (macOS)]
I recently created a virtual machine by going through the virtualize option process and then changed the architecture to x86_64. When I installed Windows 7 on my M1, I installed spice tools and then went to the drive image options to change from Spice tools to Windows 7 ISO and then shut down the virtual machine to change the display emulated card.  When I turn back on the virtual machine, it says I have to install Windows again which I have to restart the whole process. What is the cause and fix of this because I don't want to go through the long process again for the fourth time.

UTM Version: 3.0.0 Beta

OS Version: Windows 7

 Apple M1 Chip

**Comment by conath** [comment]
Why are you putting the Windows 7 ISO back in the VM? If it is already installed, just clear the CD drive and it will start from the virtual hard disk again, loading Windows 7.

**Comment by emdiso** [comment]
Hi, I am having a similar problem. Every time I start the VM, it makes me go through the installation wizard, and then when it has finished installing, it restarts and then takes me to the wizard again. A couple of times, it loaded to a Windows screen, but if I shut the VM and turned it on again, I was back at the installation wizard. I am using Windows 11 arm architecture for my M2 MacBook air. Thanks.

**Comment by ElGrorg** [comment]
I am also having the same issue where the installation is stuck in a loop. @emdiso and @808ScottyA has this been solved since this post?

**Comment by alaz-aura** [comment]
I also hit this, and @conath has provided the answer. After running your install, you can 'clear' the CD drive of the .iso file by selecting the CD/DVD drive option below. 

Upon the next restart of the VM, you'll be taken to your account sign in screen to resume. 

![Screenshot 2024-08-29 at 3 49 00 PM](https://github.com/user-attachments/assets/94465fb6-452d-4020-886c-2f9050167eb8)

**Reply by derekr06** [reply]
life saver

**Reply by Cirnoo** [reply]
ohhh, life saver
